src/ui/page/share/dataset/steps/metadataFilling/metadataFilling.tsx

```diff
@@ -352,10 +362,15 @@ export const MetadataFilling: FC = () => {
         value: O.none,
         render: (): JSX.Element => (
           <div className="okp4-dataverse-portal-share-data-metadata-filling fee" key={id10}>
-            <Field
+            <NumericField
+              decimalScale={APP_ENV.chains[0].feeCurrencies[0].coinDecimals} // 6
+              decimalSeparator="."
               id={id10}
-              label={'fee'}
+              lang="fr-FR"
```

Comment:
Should it be better to get user current language maybe?

Reply:
Makes sense! The field can display formatted number based on locale by default.